samples/minecraft/README.md

```diff
@@ -0,0 +1,5 @@
+# Minecraft GSDK sample for thundernetes
+
+Check the [quickstart](https://github.com/PlayFab/thundernetes/blob/main/docs/quickstart.md) on how to run the Minecraft sample on thundernetes.
+
+This image is based on the itzg/minecraft-server docker image, but modified to initiate the server with the GSDK wrapper, since Minecraft is not an OSS. The modified dockerfile can be found [here](https://github.com/rnjohn/docker-minecraft-server-gsdk).
```

Comment:
>docker [](http://example.com/codeflow?start=223&length=6)

Can we link to itzg/minecraft-server repository and maybe add some details on how it's working, what version of Minecraft it's based on etc.

Reply:
Yup, I added more info in the README, let me know if it's alright